### ISSUE TITLE
[Suggestion] Support for extra wifi settings

### ISSUE THREAD

**Issue by mrbostad** [meatpiHQ/wican-fw#24]
Some of us have wifi in car and in garage / home, it would be a nice feature if one could setup multiple wifi configurations so the wican obd could jump over to the house/garage wifi when the car is home and turned off / the in-car-wifi is off.

**Comment by jnimmo** [comment]
Yeah I love this idea too

**Comment by meatpiHQ** [comment]
@mrbostad @jnimmo Noted

**Comment by TommySharpNZ** [comment]
Both great suggestions!

**Comment by mrbostad** [comment]
> Both great suggestions! Although possibly we'd also need options to set different MQTT connection settings for each wifi AP?

Good point, i did not think of that since my car-routers have GRE tunnel to my home router, so my mqtt broker is always available. So yeah, maybe a "failover-mqtt-broker" option? (if mqtt1 is not answering, try mqtt2 or something like that?)
